modules/llm-cache/tests/kv_state_cache_local_file_test.cc

```diff
@@ -172,7 +173,14 @@ void inference(std::shared_ptr<KVStateCacheManager>& kv_state_cache_manager,
   }
 }
 
-void threadFunc() {
+void checkFilesNotExist(std::string dir) {
+  for (auto it = std::filesystem::recursive_directory_iterator(dir);
+       it != std::filesystem::recursive_directory_iterator(); ++it) {
+    VINEYARD_ASSERT(!std::filesystem::is_regular_file(*it));
```

Comment:
I suggest no `std::filesystem` as it requires C++17, which makes our code cannot be built without such a new C++ compiler.

We have included a thirdparty one in v6d repo, try grep `ghc::filesystem` in the codebase for how to use it.

Reply:
OK